Procfile

```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p 3000
+worker: bundle exec sidekiq --queue default
```

Comment:
What if we want to use queue other than `default`?

Reply:
Update this line

Reply:
OK.